app/presenters/hyrax/generic_work_presenter.rb

```diff
@@ -1,6 +1,6 @@
 module Hyrax
   class GenericWorkPresenter < Hyrax::WorkShowPresenter
-    delegate :displays_in, to: :solr_document
-    delegate :geographic_name, to: :solr_document
+    include Tufts::SharedTermsService
```

Comment:
I'm not sure I follow what these includes are doing? Do we want these classes to all inherit this service, or can they just call it as a collaborator?